features/channel/managing_channels/adding_channel.feature

```diff
@@ -31,6 +31,7 @@ Feature: Adding a new channel
         And I define its color as "blue"
         And I choose "Euro" as the base currency
         And I choose "English (United States)" as a default locale
+        And I allow to skip shipping step if only one shipping method is available
```

Comment:
We don't validate if this property is set, but at least check if we can select it, so it's :+1: